File: dotnet/Samples/Lps/CloudLanguagePairService.cs
using LanguageWeaver.Sdk.Common.Cloud;
using LanguageWeaver.Sdk.LanguagePairs.Cloud.Result;

namespace LanguageWeaver.Sdk.Samples.Lps;

public static class CloudLanguagePairService
{
    public static void Main()
    {
        using var lwClient = new CloudLanguageWeaverClient().Build();
        CloudLanguagePairsResult languagePairsResult = lwClient.GetCloudLanguagePairs();

        languagePairsResult.LanguagePairs
            .ForEach(languagePair => Console.WriteLine(languagePair.Name));
    }
}

File: java/src/main/java/com/languageweaver/sdk/samples/lps/CloudLanguagePairService.java
package com.languageweaver.sdk.samples.lps;

import com.languageweaver.sdk.CloudLanguageWeaverClient;
import com.languageweaver.sdk.translate.result.cloud.CloudLanguagePairsResult;

public class CloudLanguagePairService {

    public static void main(String[] args) throws Exception {
        try (CloudLanguageWeaverClient lwClient = new CloudLanguageWeaverClient().build()) {
            final CloudLanguagePairsResult languagePairsResult = lwClient.getCloudLanguagePairs();

            if (languagePairsResult.getLanguagePairs() != null) {
                languagePairsResult.getLanguagePairs()
                        .forEach(languagePair -> System.out.println(languagePair.getName()));
            }
        }
    }
}
